src/privatesend/privatesend-client.cpp

```diff
@@ -20,6 +20,10 @@
 
 #include <memory>
 
+#ifdef ENABLE_WALLET
+#include "wallet/wallet.h"
```

Comment:
`wallet.h` is already included in `privatesend-client.h`, should keep one or the other, not both imo. Also, there is no need for `#ifdef` because `privatesend-client.cpp` is only compiled and linked when wallet is enabled in `configure`(can't mix on the client-side without wallet).

Reply:
dropped wallet.h then